app/Assets/SizeVariantGroupedWithRandomSuffixNamingStrategy.php

```diff
@@ -156,11 +156,11 @@ protected static function createRndMiddlePath(): string
 			$rndStr = bin2hex(random_bytes(self::NAME_LENGTH / 2));
 
 			return
-				\Safe\substr($rndStr, 0, 2) .
+				substr($rndStr, 0, 2) .
```

Comment:
What happened to the safe variant? Here and below?

Reply:
https://github.com/LycheeOrg/Lychee/pull/1469#discussion_r983842184